HacktoberfestProject.Web/Models/DTOs/User.cs

```diff
@@ -10,7 +10,7 @@ public class User
 
 		public User(string username, List<Repository> repositoryPrAddedTo = null)
 		{
-			RepositoryPrAddedTo = repositoryPrAddedTo;
+			RepositoryPrAddedTo = repositoryPrAddedTo ?? new List<Repository>();

```

Comment:
Use the static class NullChecker to perform the nullcheck

Reply:
Let's clarify this a bit. When I added this it was due to an empty list or repositories throwing a null exception on the view when iterating through results. I don't think we want to throw a null exception here if `repositoryPrAddedTo` is null as it is an optional argument that can be null. However initializing to an empty list prevents the aforementioned issue.